src/dll/Hooks/ReportErrorCode.cpp

```diff
@@ -0,0 +1,73 @@
+#include "ReportErrorCode.hpp"
+#include "Addresses.hpp"
+#include "App.hpp"
+#include "Hook.hpp"
+#include "stdafx.hpp"
+
+namespace
+{
+bool isAttached = false;
+
+void _State_SessionActive_ReportErrorCode(uintptr_t);
+Hook<decltype(&_State_SessionActive_ReportErrorCode)> State_SessionActive_ReportErrorCode(
+    0x7FA31576, &_State_SessionActive_ReportErrorCode);
+
+void _State_SessionActive_ReportErrorCode(uintptr_t aThis)
```

Comment:
Same as the file's comment, the name is a bit ambiguous. I would prefer something like `CLASS/SYSTEM_FUNCNAME`.